2018/python/rules_engine.py

```diff
@@ -305,31 +305,34 @@ def s305(self):
 		fail_df = self.lar_df[self.lar_df.duplicated(keep=False)==True] #pull frame of duplicates
 		self.results_wrapper(edit_name=edit_name, field_name=field, fail_df=fail_df)
 
-	def v608(self):
+	def v608_1(self):
 		"""A ULI with an invalid format was provided.
 		1) The required format for ULI is alphanumeric with at least 23 characters and up to 45 characters, 
 		and it cannot be left blank.
-		Exempt Filers:
-		If your institution is reporting a Loan/Application Number, the required format for Loan/Application Number 
-		is alphanumeric with at least 1 character and no more than 22 characters, and it cannot be left blank."""
-
-		edit_name = "v608"
+		"""
+		edit_name = "v608_1"
 		field = "ULI"
-		#if length not between 23 and 45 or if ULI is blank
-		#get subset of LAR dataframe that fails ULI conditions
-		#check if LEI present as first 20 digits of ULI
+		#Obtaining subset of LAR that have ULIs rather than NULIs.
```

Comment:
Can you move your comment so that it is above line 317 instead of 316?

Reply:
Sure!